### PR TITLE
fix docs

### LINKED ISSUE #12120: torch.argsort not showing up in docs

### PR DESCRIPTION
**Pull request by weiyangfb**
- fix https://github.com/pytorch/pytorch/issues/12120
- add `torch.argsort`, `torch.pdist`, `broadcast_tensors` to *.rst files
- add parameter dim to `torch.unique` doc
- fix table and args for `torch.norm`
- test plan: make html and check docs in browser

@gchanan

### CODE REVIEW

File: docs/source/torch.rst [hunk 1]
```diff
@@ -270,6 +270,8 @@ Other Operations
 .. autofunction:: trace
 .. autofunction:: tril
 .. autofunction:: triu
+.. autofunction:: argsort
```
Comment (vishwakftw) [inline docs/source/torch.rst]: Shouldn’t `argsort` be in the same category as `sort`? And could you please add this in alphabetical order?

**Review by gchanan — APPROVED** [review]

File: torch/_torch_docs.py [hunk 2]
```diff
@@ -1210,7 +1210,7 @@ def parse_kwargs(desc):
 Computes the logarithmic derivative of the gamma function on `input`.
 
 .. math::
-    \psi(x) = \frac{d}{dx} \ln\left(\Gamma\left(x\right)\right) = \frac{\Gamma'(x)}{\Gamma(x)}
+    \psi(x) = \frac{d}{dx} \ln\left(\gamma\left(x\right)\right) = \frac{\gamma'(x)}{\gamma(x)}
```
Comment (vishwakftw) [inline torch/_torch_docs.py]: @weiyangfb Why was this changed?

Comment (weiyangfb) [reply torch/_torch_docs.py]: `\Gamma` doesn't show up correctly

**Comment by SsnL** [comment]
\Gamma and \gamma are two different things! Please revert that change.

**Comment by weiyangfb** [comment]
I see, will change it back
